src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -219,4 +219,10 @@
   <data name="SARIF1020_SchemaReferenceMissing" xml:space="preserve">
     <value>{0}: The SARIF log file does not contain a $schema property. Add a $schema property that refers to the final version of the SARIF 2.1.0 schema. This enables IDEs to provide Intellisense for SARIF log files.</value>
   </data>
+  <data name="SARIF1021_Default" xml:space="preserve">
+    <value>{0}: The tool driver version is missing.</value>
```

Comment:
>. [](start = 50, length = 1)

We can include dynamic data:
`{0}: The description of the tool '{1}' does not include a 'version' property. The 'version' property is required in workflows that must verify the version of the tool being used."

Reply:
nice! thanks!

---
In reply to: [434751150](https://github.com/microsoft/sarif-sdk/pull/1899#discussion_r434751150) [](ancestors = 434751150)